src/passwordcheck.js

```diff
@@ -27,12 +27,26 @@ define([
       PASSWORD_TOO_SHORT: 'PASSWORD_TOO_SHORT'
     };
 
-    function isStrongEnough(password) {
-      // Check for passwords that at least contain a number and an alphabet,
-      // or if alphabets, then at least (minLength + 4) characters long
-      var regexString = '((?=.*[a-zA-Z])(?=.*[0-9])[A-Za-z0-9!@#$%^&*()_+ ]{' + minLength + ',' + (minLength + 4) + '})|([A-Za-z0-9!@#$%^&*()_+ ]{' + (minLength + 4) + ',})';
```

Comment:
@TDA - If we make this change, will the bloom filter data set need to be updated?